app/assets/stylesheets/components/_reveal.scss

```diff
@@ -6,6 +6,15 @@
   padding: .2em .5em .3em .7em;
 }
 
+// this is used in places where we want the reveal to match the width of the form inputs above it
+.reveal-shrink-wrapper {
+  .reveal {
+    @media screen and (min-width: $mobile-up) {
+      width: 475px;
```

Comment:
it seems like this is trying to match the width of classes like `.form-width--long`, which is [actually defined as `$width-form-long` or 25em](https://github.com/codeforamerica/honeycrisp-gem/blob/main/app/assets/stylesheets/honeycrisp/atoms/_form-widths.scss#L10)

so we should probably put `$width-form-long` here instead

Reply:
`.form-width--long` also defines it as a `max-width` instead of a `width`, which is helpful for a small range of screen sizes (between `$mobile-up` (=481px) and ~500px)

![Screen Shot 2021-12-14 at 11 25 02 AM](https://user-images.githubusercontent.com/30675659/146065963-a1368e1a-39e4-4589-8fc2-b5d7d6be3bf9.png)

it's possible if we set it as `max-width` we can drop the media query

Reply:
oh wow that's much better